sdk/python/Makefile

```diff
@@ -67,7 +69,6 @@ test_all:: test_fast test_auto test_go
 
 PULUMI_TEST_ORG ?= $(shell pulumi whoami --json | jq ".organizations[0]")
 
-dist:: GOBIN=$(or $(shell go env GOBIN),$(shell go env GOPATH)/bin)
 dist::
 	GOBIN=${GOBIN} go install -C cmd/pulumi-language-python \
```

Comment:
Is the GOBIN for L73 set elsewhere, or should `GOBIN=${GOBIN}` be removed here?

Reply:
Hrm if you remove line 70, does running `make dist` in `sdk/python` still work?  Not that I think anyone is running that :thinking: 

So it's probably fine?

Reply:
With `mise`, I think `$GOBIN` is always set, but without mise it's not.

`go help install` says:

> Executables are installed in the directory named by the GOBIN environment
> variable, which defaults to $GOPATH/bin or $HOME/go/bin if the GOPATH
> environment variable is not set. Executables in $GOROOT
> are installed in $GOROOT/bin or $GOTOOLDIR instead of $GOBIN.

So I think line 70 is moot?

Reply:
Ah I guess this was an issue in a place where we didn't use `go install` maybe :thinking: 

I think I ran into having to set this because `go env GOBIN` doesn't give you the same thing as `go install` does:

```
$ go env GOBIN                                  

$ go env GOPATH
/home/tgummerer/go
```

But I think you can still remove `GOBIN=${GOBIN}` as @iwahbe suggests then.